source/PlanetPanel.cpp

```diff
@@ -110,17 +111,22 @@ void PlanetPanel::Draw()
 
 	if(planet.CanUseServices())
 	{
-		if(planet.IsInhabited())
-		{
-			info.SetCondition("is inhabited");
+		const Port &port = planet.GetPort();
+
+		if(port.HasService(Port::ServicesType::Bank))
 			info.SetCondition("has bank");
-			if(system.HasTrade())
-				info.SetCondition("has trade");
+		if(port.HasService(Port::ServicesType::JobBoard))
+			info.SetCondition("has job board");
+		if(port.HasService(Port::ServicesType::HireCrew))
+			info.SetCondition("can hire crew");
+		if(port.HasService(Port::ServicesType::Trading) && system.HasTrade())
+			info.SetCondition("has trade");
+		if(planet.HasNamedPort())
+		{
+			info.SetCondition("has port");
```

Comment:
we used to set the "has spaceport" condition, shouldnt we set both?
or mb a spaceport is when it offers all the services, like before?

Reply:
ik but that means we dont have the old one anymore, and since it used to mean all services were there it could be a quick way to say you wnt that as the destination of a mission for instance

Reply:
Info conditions aren't the same as player conditions. Info conditions don't get can't be missions.